test/views/LockedViewSpec.scala

```diff
@@ -38,7 +38,7 @@ class LockedViewSpec extends ViewBehaviours {
   behave like pageWithContent("p", "Another person from your organisation is currently working on this declaration. You cannot open it until they’re finished.")
 
   s"must render button" in {
-    val button = getElementByClass(doc, "govuk-button")
+    val button = doc.getElementsByClass("govuk-button").last()
     assertElementContainsText(button, "Return to drafts")
     assertElementContainsHref(button, frontendAppConfig.manageTransitMovementsDraftDeparturesUrl)
   }
```

Comment:
This could be
```
behave like pageWithButton(doc, "Return to drafts) {
  assertElementContainsHref(_, frontendAppConfig.manageTransitMovementsDraftDeparturesUrl)
}
```